### PR TITLE
Feature/messenger mission

### PR DESCRIPTION
**Pull request by micahnyc**
messenger and mercury assets for messenger scene

### CODE REVIEW

File: data/assets/messenger.scene [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+asset.request('scene/digitaluniverse/stars')
+asset.request('scene/digitaluniverse/milkyway')
+
+asset.require('util/default_keybindings')
+asset.require('util/default_dashboard')
+
+local MessengerAsset = asset.require('scene/solarsystem/missions/messenger/messengerSC')
+
+if not openspace.modules.isLoaded("Volume") then
+    openspace.printErro("Volume module is not loaded, skipping asset: mercurymagnetosphere")
```
Comment (alexanderbock) [inline data/assets/messenger.scene]: `printError`

File: data/assets/messenger.scene [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+else
+    asset.require('scene/solarsystem/missions/messenger/mercurymagnetosphere')
+end
+
+asset.request('customization/globebrowsing')
+
+assetHelper.registerDashboardItems(asset, {
+    {
+        Type = "DashboardItemDistance",
+        Identifier = "Messenger1Distance",
```
Comment (alexanderbock) [inline data/assets/messenger.scene]: Extra 1

File: data/assets/scene/solarsystem/missions/messenger/mercurymagnetosphere.asset [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    Transform = {
+        Translation = {
+            Type = "SpiceTranslation",
+            Target = "MERCURY",
+            Observer = "MERCURY BARYCENTER",
+        },
+        Rotation = {
+            Type = "SpiceRotation",
+            SourceFrame = "MERCURYSE",
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/missions/messenger/mercurymagnetosphere.asset]: SE is correct?

File: data/assets/scene/solarsystem/missions/messenger/messengerSC.asset [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    kernels .. '/pck/pck00008.tpc',
+    kernels .. '/pck/pck00008_msgr.tpc',
+    kernels .. '/pck/pck00009_msgr_v10.tpc',
+    kernels .. '/pck/pck00010_msgr_v10.tpc',
+    kernels .. '/pck/pck00010_msgr_v23.tpc',
+    kernels .. '/pck/pck00010.tpc'
+}
+
+
+RotationMatrix = {
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/missions/messenger/messengerSC.asset]: `local RotationMatrix`

File: data/assets/scene/solarsystem/missions/messenger/messengerSC.asset [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+        ColorTexture = models .. "/foil_n2.png",
+        ModelTransform = RotationMatrix
+    },
+    GUI = {
+        Name = "MessengerProbe foil",
+        Path = "/Solar System/Missions/Messenger"
+    }
+}
+
+local MessengerProbeHeatSheild = {
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/missions/messenger/messengerSC.asset]: `Shield`

File: data/assets/scene/solarsystem/missions/messenger/messengerSC.asset [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+        Path = "/Solar System/Missions/Messenger"
+    }
+}
+
+
+assetHelper.registerSceneGraphNodesAndExport(asset, {
+    Messenger,
+    MessengerProbeBlack,
+    MessengerProbeFoil,
+    MessengerProbeHeatSheild,
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/missions/messenger/messengerSC.asset]: `Shield`

File: data/assets/scene/solarsystem/missions/messenger/mercurymagnetosphere.asset (hunk 3 above)
Comment (alexanderbock) [reply data/assets/scene/solarsystem/missions/messenger/mercurymagnetosphere.asset]: Nevermind, just saw it

Comment (micahnyc) [reply data/assets/scene/solarsystem/missions/messenger/mercurymagnetosphere.asset]: well, I guess in that the magnetosphere had the correct orientation in regards to the planet and the sun. and none of the science people said anything to the contrary.

**Pull request merged**